Alfie/Alfie/Delegate/AppDelegate.swift

```diff
@@ -54,7 +54,14 @@ class AppDelegate: NSObject, UIApplicationDelegate, UNUserNotificationCenterDele
                 serviceProvider.notificationsService.start()
             }
         }
-        tabCoordinator = TabCoordinator(tabs: TabScreen.allCases, activeTab: .home(), serviceProvider: serviceProvider)
+
+        var tabs: [TabScreen] = [.home(nil), .shop(tab: nil), .bag]
```

Comment:
Home and shop cases have default values for parameters, you can skip adding them:
```suggestion
        var tabs: [TabScreen] = [.home(), .shop(), .bag]
```